src/middlewared/middlewared/plugins/enclosure_/m50_plx.py

```diff
@@ -10,11 +10,12 @@ class EnclosureService(Service):
 
     @private
     def m50_plx_enclosures(self):
-        system_product = self.middleware.call_sync("system.info")["system_product"]
-        if system_product is None or not ("TRUENAS-M50" in system_product or "TRUENAS-M60" in system_product):
+        """
```

Comment:
@rick-mesta yeah, the `m50_plx_enclsoures` method makes nvme related calls that don't exist on SCALE HA, yet. So that entire method is a NO-OP for now. I just changed the code there to match the other changes in this PR.